app/src/main/java/ch/epfl/sweng/erpa/CreateGame.java

```diff
@@ -1,13 +1,141 @@
 package ch.epfl.sweng.erpa;
 
-import android.app.Activity;
+import android.app.AlertDialog;
+import android.content.DialogInterface;
+import android.content.Intent;
+import android.graphics.Color;
+import android.net.Uri;
 import android.os.Bundle;
+import android.support.annotation.IdRes;
+import android.support.v7.app.AppCompatActivity;
+import android.util.Log;
+import android.view.View;
+import android.widget.AdapterView;
+import android.widget.EditText;
+import android.widget.RadioButton;
+import android.widget.Spinner;
+import android.widget.TextView;
 
-public class CreateGame extends Activity {
+import butterknife.BindView;
+import butterknife.ButterKnife;
+import butterknife.OnClick;
+import ch.epfl.sweng.erpa.model.Game;
+
+public class CreateGame extends AppCompatActivity implements BlankFragment.OnFragmentInteractionListener {
 
     @Override
     protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
         setContentView(R.layout.activity_create_game);
+        ButterKnife.bind(this);
+        final EditText universeField = findViewById(R.id.universe_field);
+        final Spinner universesSpinner = findViewById(R.id.universes_spinner);
+        universesSpinner.setOnItemSelectedListener(new AdapterView.OnItemSelectedListener() {
+            @Override
+            public void onItemSelected(AdapterView<?> parentView, View selectedItemView, int position, long id) {
+                String feedbackType = universesSpinner.getSelectedItem().toString();
+                if(feedbackType.equals("Other"))
+                    universeField.setVisibility(View.VISIBLE);
+                else
+                    universeField.setVisibility(View.GONE);
+            }
+            @Override
+            public void onNothingSelected(AdapterView<?> parentView) {}
+        });
+    }
+
+    @Override
+    public void onFragmentInteraction(Uri uri) {
+        Log.i("coucou",uri.toString());
```

Comment:
“Coucou” is not an acceptable tag, please use a more descriptive prefix or remove this line.